meshtastic/__main__.py

```
@@ -1364,105 +1818,125 @@ def common():
+                    if client.devPath is None:
+                        try:
+                            client = stack.enter_context(
+                                meshtastic.tcp_interface.TCPInterface(
+                                    "localhost",
+                                    debugOut=logfile,
+                                    noProto=args.noproto,
+                                    noNodes=args.no_nodes,
+                                    timeout=args.timeout,
+                                )
+                            )
+                        except Exception as ex:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)